dailyjournal.sql

```diff
@@ -4,21 +4,21 @@ CREATE TABLE `Entries` (
   `entry` TEXT NOT NULL,
   `date` INTEGER NOT NULL,
   `moodId` INTEGER NOT NULL,
-  FOREIGN KEY(`moodId`) REFERENCES `Moods`(`id`),
+  FOREIGN KEY(`moodId`) REFERENCES `Moods`(`id`)
 );
 
 CREATE TABLE `Moods` (
   `id`	INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
   `label` TEXT NOT NULL,
 );
 
-INSERT INTO `Entries` VALUES(null, "12345", "123", 1598458543321, 1)
-INSERT INTO `Entries` VALUES(null, "abc", "123", 1598458548239, 2, 2)
-INSERT INTO `Entries` VALUES(null, "Delete", "Now Deleting", 1598458559152, 1, 3)
-INSERT INTO `Entries` VALUES(null, "ANGRY", "jij", 1598557358781, 3, 4)
-INSERT INTO `Entries` VALUES(null, "678", "Now Deleting", 1598557373697, 4, 5)
+INSERT INTO `Entries` VALUES (null, "12345", "123", 1598458543321, 1);
+INSERT INTO `Entries` VALUES(null, "abc", "123", 1598458548239, 2, 2);
+INSERT INTO `Entries` VALUES(null, "Delete", "Now Deleting", 1598458559152, 1, 3);
+INSERT INTO `Entries` VALUES(null, "ANGRY", "jij", 1598557358781, 3, 4);
+INSERT INTO `Entries` VALUES(null, "678", "Now Deleting", 1598557373697, 4, 5);
 
-INSERT INTO `Moods` VALUES(null, "Happy")
-INSERT INTO `Moods` VALUES(null, "Sad")
-INSERT INTO `Moods` VALUES(null, "Angry")
-INSERT INTO `Moods` VALUES(null, "Ok")
+INSERT INTO `Moods` VALUES(null, "Happy");
+INSERT INTO `Moods` VALUES(null, "Sad");
+INSERT INTO `Moods` VALUES(null, "Angry");
+INSERT INTO `Moods` VALUES(null, "Ok");
```

Comment:
You are setting the id to null, which is not possible, because in the table you set the id field as 'NOT NULL'.

Notice the 'AUTOINCREMENT' command on the id values?  That means that they will be automatically assigned a value.  So you can actually just remove the 'null' values in your INSERT statements.

To test, you can use the command `select * from entries;` in your `.sql` file to get check if that data is populated.